### PR TITLE
Bear - fixes 861

### PR DESCRIPTION
**Pull request by ron-myers**
## Description

Fixes #861 where extension would stay open after completing a task.

Updates flows for:
* Create Note
* Add Text
* Web Capture
* Grab Note Url

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @ron-myers 👋 

You should be able to remove it from the PR, since it makes it impossible to track and if the other PR takes more time to close, then this stall.

Can you look into this issue, let me know if you need any help doing this 💪 

Besides this issue, @hmarr, do you have any feedback?

**Comment by ron-myers** [comment]
Fair point on the removal @pernielsentikaer .  That commit is removed.

**Comment by pernielsentikaer** [comment]
Awaiting feedback until tomorrow by @hmarr

File: extensions/bear/src/note-actions.tsx [hunk 1]
```diff
@@ -43,19 +44,25 @@ function NotePreviewAction({ note }: { note: Note }) {
...
+};
+
 export default function NoteActions({ isNotePreview, note }: { isNotePreview: boolean; note: Note }) {
   return (
     <ActionPanel>
       <ActionPanel.Section title="Open">
         <OpenAction
           title="Open in Bear"
           target={`bear://x-callback-url/open-note?id=${note.id}&edit=yes`}
+          onOpen={onBrowserOpen}
```
Comment (mathieudutour) [inline extensions/bear/src/note-actions.tsx:58]: That's deviating from the standard Raycast behaviour. Raycast will automatically pop to root after a while depending on the user's settings and we assume that if they open raycast again during that lapse, we should keep the context they were in.
While it makes sense to pop to root for the `create note` screen, I don't know if that's what we should do here. What do you think?

Comment (ron-myers) [reply extensions/bear/src/note-actions.tsx:58]: Hey @mathieudutour that is a good discussion point, that I feel is blurry currently within Raycast extensions.  I am open to changing where it is used in this PR.  

When reviewing I went with:
1. my gut on what felt right to always pop to the root, I used the filter of `After I execute this action, will I want to come back to this state or move on to another task.  Especially quickly.
2. reviewed [existing extension behavior](https://github.com/raycast/extensions/search?q=poptoroot)

Is the standard behaviour written down so I can follow it moving forward - apologies if I missed that in the docs.

Comment (mathieudutour) [reply extensions/bear/src/note-actions.tsx:58]: >  I feel is blurry currently within Raycast extensions

That's a good point, we haven't really written down guidelines for that yet - please *bear* with us (pun intended 😬).

>  I used the filter of `After I execute this action, will I want to come back to this state or move on to another task. Especially quickly.

I do think that's a good filter, but would argue that when you are looking for a note in Bear using the search, you might actually need to review a few of them before finding the right one? I'm not sure. But I'll defer to what you think is best while we think internally about a more standard behaviour.

Comment (ron-myers) [reply extensions/bear/src/note-actions.tsx]: I followed your gut on this.  If it is in the way while using it - I will put in another PR.  

Thanks for your review.

**Pull request merged**